### PR TITLE
Simplify package structure

### PR DESCRIPTION
**Pull request by jhunkeler**
TODO:
- [x] fix docs

cc @jdavies-st

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,161 +21,160 @@ sudo: false
...
-        # E112 - 4 spaces per indentation level
-        # E113 - 4 spaces per indentation level
-        # E502 - the backslash is redundant between brackets
-        # E722 - do not use bare except
-        # E901 - SyntaxError or IndentationError
-        # E902 - IOError
-        - FLAKE8_OPT="--select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902"
+      # Set defaults to avoid repeating in most cases
+      - PYTHON_VERSION=3.6
+	    - NUMPY_VERSION=stable
```
Comment (jdavies-st) [inline .travis.yml]: `tab`

**Comment by jdavies-st** [comment]
Fantastic.  Merge, tag and release as you see fit.

**Comment by mcara** [comment]
I concur: Fantastic! Thank you @jhunkeler! 🥇

**Comment by mcara** [comment]
Screen Shot 2019-06-27 at 3 22 47 PM

**Pull request merged**